raiden/tests/benchmark/conftest.py

```diff
@@ -0,0 +1,90 @@
+import contextlib
+import shlex
+import subprocess
+import time
+
+import pytest
+
+from raiden.tests.integration.fixtures.blockchain import *  # noqa: F401,F403
+from raiden.tests.integration.fixtures.raiden_network import *  # noqa: F401,F403
+from raiden.tests.integration.fixtures.smartcontracts import *  # noqa: F401,F403
+from raiden.tests.integration.fixtures.transport import *  # noqa: F401,F403
+
+from . import _codespeed
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--codespeed-url",
+        action="store",
+        default=None,
+        help="the URL of a running Codespeed instance, " "e.g. http://127.0.0.1:8000",
+    )
+    parser.addoption(
+        "--output-benchmark-results",
+        action="store",
+        default=None,
+        help="the path to write benchmark results to",
+    )
+
+
+def pytest_collection_modifyitems(items):
+    for item in items:
+        item.add_marker(pytest.mark.asyncio)
+
+
+_TIMES = {}
+
+
+_COMMIT_ID = (
```

Comment:
Should this be done on import time? It could be placed into a session scoped `bench` fixture. But I don't see any specific problem with doing it on import time, either.